src/main/resources/application-docker.yml

```diff
@@ -0,0 +1,15 @@
+spring:
+
+  flyway:
```

Comment:
Does flyaway automatically run migrations when docker spins up?

Reply:
Yes, so if the database exists it does nothing, if it doesn't it'll create it and run the data inserts in /migration/docker which is the main difference with this profile.